portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +34,98 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+private List<String> comments;
+
+  @Override
+  public void init() {
+    comments = new ArrayList<>();
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();    
+    PreparedQuery results = datastore.prepare(query);
+    List<Comment> commentsList = new ArrayList<>();
```

Comment:
[nit (short for nitpicky comment)] I would just choose List or ArrayList for both the left and right side, that way you can only import one of the two